spec/user_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+# require_relative '../app/models/user.rb'
```

Comment:
- [ ] Please kindly get rid of commented code for it makes your code base look untidy and messy.